obspy/io/resp/nrl.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+nrl.py for navigating, and picking responses from the
+Nominal Response Library.
+
+:copyright:
+    Lloyd Carothers IRIS/PASSCAL, 2016
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+'''
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+# Urllib for py 2 and 3
+from future.standard_library import install_aliases
+install_aliases()  # NOQA
```

Comment:
Can you please avoid using `future.standard_library`? It has been causing problems (although not for urllib imports but for Queue). Please see #1614 for how to do these imports without future.